Controllers/UserController.cs

```diff
@@ -48,25 +43,19 @@ public async Task<IActionResult> Login(LoginDTO model)
             if (user == null)
                 return ApiBadRequest("User does not exist.");
 
-            var result =
-                await _signInManager.PasswordSignInAsync(user.UserName, model.Password, false, lockoutOnFailure: false);
-            if (result.IsLockedOut)
-                return ApiBadRequest("User account locked out.");
-
-            if (!result.Succeeded)
-                return ApiBadRequest("Invalid username or password.");
-
-            var roles = await _userManager.GetRolesAsync(user);
-
-            return Ok(new
-            {
-                token = _jwt.GenerateSecurityToken(new JwtUser()
+            if (user != null && await _userManager.CheckPasswordAsync(user, model.Password)) {
```

Comment:
We dont need `user != null` check here because check above ensures user is not `null` anyway.